docs/spec/normative_examples.md

```diff
@@ -504,68 +507,96 @@ status:
+                         abc   2018-01-17 10:32    user1        33643fc
 
-$ knative rollout next percent 5
+$ knative release --service my-service begin ghi
+
+$ knative revisions list --service my-service
+Route           Traffic  Id    Date                Deployer     Git SHA
+next,latest     0%       ghi   2018-01-19 12:16    user1        a6f92d1
+current         100%     def   2018-01-18 20:34    user1        a6f92d1
+                         abc   2018-01-17 10:32    user1        33643fc
+
+$ knative release percent 5
 [...]
-$ knative rollout next percent 50
+
+$ knative revisions list --service my-service
+Route           Traffic  Id    Date                Deployer     Git SHA
+next,latest     5%       ghi   2018-01-19 12:16    user1        a6f92d1
+current         100%     def   2018-01-18 20:34    user1        a6f92d1
+                         abc   2018-01-17 10:32    user1        33643fc
+
+$ knative release --service my-service percent 50
 [...]
-$ knative rollout finish
+$ knative release --service my-service finish
 [...]
 
 $ knative revisions list --service my-service
-Name          Traffic  Id   Date                Deployer      Git SHA
-current,next  100%     v3   2018-01-19 12:16    user1         a6f92d1
-                       v2   2018-01-18 20:34    user1         a6f92d1
-                       v1   2018-01-17 10:32    user1         33643fc
+Route           Traffic  Id    Date                Deployer      Git SHA
+current,latest  100%     ghi   2018-01-19 12:16    user1         a6f92d1
+                         def   2018-01-18 20:34    user1         a6f92d1
+                         abc   2018-01-17 10:32    user1         33643fc
 ```
 
 **Steps**:
 
-* Update the Service to switch from `runLatest` to `pinned` strategy.
+* Update the Service to switch from a `runLatest` strategy to a `release`
+  strategy.
 
 * Update the Service with the new configuration (env var).
 
-* Update the Service to address the new Revision.
+* Update the Service include the new revision in its revision list, which makes
+  it address the new Revision as `next`.
+
+* Adjust the `percentRollout` controlling the traffic on `next`.
+
+* Complete the rollout so the new revision is `current`.
 
 **Results:**
 
-* The system creates the new revision from the configuration,
-  addressable at next.my-service... (by convention), but traffic is
-  not routed to it until the percentage is manually ramped up. Upon
-  completing the rollout, the next revision is now the current
-  revision.
+* The system creates the new revision from the configuration, addressable at
+  `latest.my-service...`, but traffic is not routed to it until the rollout to
+  that revision is begun (which marks it as `next.my-service...`) and the
+  percentage is manually ramped up. Upon completing the rollout, the next
+  revision is now the current revision.
 
-![Manual rollout](images/manual_rollout.png)
+![Release mode](images/manual_rollout.png)
```

Comment:
'manual' here seems confusing with the addition of 'manual' type that removes the Service from control of the underlying Configuration and Route.

Reply:
It was just the name of the image left over. Changed taht.